testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/AbstractClientPoliciesTest.java

```diff
@@ -558,195 +803,843 @@ protected void deleteClientDynamically(String clientId) throws ClientRegistratio
     }
 
-    // register created component to the policy
-    protected void registerCondition(String conditionName, String policyName) {
-        ComponentRepresentation policy = getPolicy(policyName);
-        List<String> conditionIds = policy.getConfig().get(DefaultClientPolicyProviderFactory.CONDITION_IDS);
-        if (conditionIds == null) conditionIds = new ArrayList<String>();
-        ComponentRepresentation condition = getCondition(conditionName);
-        conditionIds.add(condition.getId());
-        policy.getConfig().put(DefaultClientPolicyProviderFactory.CONDITION_IDS, conditionIds);
-        updatePolicy(policy);
+    protected Object createSecureSessionEnforceExecutorConfig() {
+        return new SecureSessionEnforceExecutor.Configuration();
     }
 
-    protected ComponentRepresentation getCondition(String name) {
-        return getComponent(name, ClientPolicyConditionProvider.class.getName());
+    protected Object createSecureSigningAlgorithmEnforceExecutorConfig() {
+        return new SecureSigningAlgorithmEnforceExecutor.Configuration();
     }
 
-    protected void updateCondition(String name, Consumer<ComponentRepresentation> op) {
-        ComponentRepresentation condition = getCondition(name);
-        op.accept(condition);
-        updateComponent(condition);
+    protected Object createSecureSigningAlgorithmForSignedJwtEnforceExecutorConfig() {
+        return new SecureSigningAlgorithmForSignedJwtEnforceExecutor.Configuration();
     }
 
-    protected void deleteCondition(String conditionName, String policyName) {
-        ComponentRepresentation policy = getPolicy(policyName);
-        List<String> conditionIds = policy.getConfig().get(DefaultClientPolicyProviderFactory.CONDITION_IDS);
-        ComponentRepresentation condition = getCondition(conditionName);
-        String conditionId = condition.getId();
-        adminClient.realm(REALM_NAME).components().component(conditionId).remove();
-        conditionIds.remove(conditionId);
-        policy.getConfig().put(DefaultClientPolicyProviderFactory.CONDITION_IDS, conditionIds);
-        updatePolicy(policy);
+    // Client Policies CRUD Operation
+
+    protected static class ClientPoliciesBuilder {
+        private final ClientPoliciesRepresentation policiesRep;
+
+        public ClientPoliciesBuilder() {
+            policiesRep = new ClientPoliciesRepresentation();
+            policiesRep.setPolicies(new ArrayList<>());
+        }
+
+        public ClientPoliciesBuilder addPolicy(ClientPolicyRepresentation rep) {
+            policiesRep.getPolicies().add(rep);
+            return this;
+        }
+
+        public ClientPoliciesRepresentation toRepresentation() {
+            return policiesRep;
+        }
+
+        public String toString() {
+            ObjectMapper mapper = new ObjectMapper();
+            String policiesJson = null;
+            try {
+                policiesJson = mapper.writeValueAsString(policiesRep);
```

Comment:
As pointed on some other places, this can be simplified to use `JsonSerialization.writeValueAsString(profilesRep);` . No need to create objectMapper for each invocation of this method. Same for ClientPoliciesBuilder and other places in this PR.

Reply:
@mposolda I see. I will try to replace codes that generate and use `ObjectMapper` with the use of existing `JsonSerialization` entirely.

Reply:
Thanks